mdf_matio/adapters/basic_adapters.py

```diff
@@ -0,0 +1,60 @@
+from mdf_matio.adapters.generic import GenericMDFAdapter
+
+
+# Adapters that require no extra processing and can just use the GenericMDFAdapter
+
+class CrystalStructureAdapter(GenericMDFAdapter):
+    """Adapt the CrystalStructureExtractor"""
+    # No extra processing required
+    pass
+
+
+class ElectronMicroscopyAdapter(GenericMDFAdapter):
+    """Adapt the ElectronMicroscopyExtractor"""
+    # No extra processing required
+    pass
+
+
+class GenericFileAdapter(GenericMDFAdapter):
+    """Adapt the GenericFileExtractor"""
+    # No extra processing required
+    pass
+
+
+class FilenameAdapter(GenericMDFAdapter):
+    """Adapt the FilenameExtractor"""
+    # No extra processing required
+    pass
+
+
+class ImageAdapter(GenericMDFAdapter):
+    """Adapt the ImageExtractor"""
+    # No extra processing required
+    pass
+
+
+class JSONAdapter(GenericMDFAdapter):
```

Comment:
Is this different than the existing JSON parser? If so, let's pick which one we want to keep

Reply:
I believe it is different, yes. The `JSONAdapter` is the adapter for the MDF-contributed `JSONParser` in MatIO. The existing code seems to also be an extractor? I'm unsure what purpose it serves, but I believe the case of extracting and adapting JSON to be served by the `JSONParser` and `JSONAdapter`.

Reply:
I think we've sufficiently discussed this one :smiley: